docs/naps/7-key-binding-dispatch.md

```diff
@@ -50,21 +50,30 @@ a **key binding** binds a key sequence to a command with conditional activation
 
 ### Key binding validity: convenience vs. complexity
 
-Some users want to use traditional modifier keys as a base key in key binding for convenience purposes [^id2]. However, this can lead to conflicts since many key bindings may include the modifier key in their key sequence and thus cause confusion and cost extra engineering effort.
+Some users want to use traditional modifier keys as a 'base key' in key binding for convenience purposes [^id2]. However, this can lead to conflicts since many key bindings may include the modifier key in their key sequence and thus cause confusion and cost extra engineering effort.
 
-The proposed restrictions on what key sequences can be used in a key binding aim to allow for the simplest user need while cutting down on any unnecessary complexities:
+We propose that:
 
-- key combinations containing any modifier keys as a base key are invalid
-- key chords cannot contain a base key that is a modifier key (aka a single modifier)
+- key combinations can either contain one modifier key (e.g., `alt`) or any number of
+  modifier keys plus a base key (e.g., `alt+shift+v`).
+  Multiple modifier keys (e.g., `alt+shift`) are not allowed.
+- a key chord part cannot be an invalid key combinations nor a single modifier.
+    - `alt t` is **invalid** because the first part is a single modifier (even though
+      it is a valid key combination)
+    - `ctrl+x alt` is **invalid** because the second part is a single modifier
+    - `ctrl+x alt+v` is **valid**
+    - `meta meta` is **invalid** because both parts are single modifiers
 
-Here are some examples:
-- `alt` is **valid** as a base key because it contains no other modifiers and no other parts
-- `alt+meta` is **invalid** because it is a key combination comprised of only modifiers
-- `alt+t` is **valid** as a key combination
-- `alt t` is **invalid** because it is a key chord whose first part is a single modifier
-- `ctrl+x alt` is **invalid** because it is a key chord whose second part is a single modifier
-- `ctrl+x alt+v` is **valid** as a key chord
-- `meta meta` is **invalid** because it is a key chord comprised of only single modifier parts
+The proposal restricts modifier keys being used without a base key, except in the case
+of a single modifier key used in isolation.
```

Comment:
I thought I remembered in the original discussion (though I'm struggling to find comments now), that while a PLUGIN shouldn't be allowed to set a single modifier key as a keybinding an END USER should be able to do whatever they please. This is a concept we've discussed also in the context of menu contributions (a plugin shouldn't be able to add/change top level menus, but a user can do whatever), and I think it would be good to mention it in this NAP also... But curious whether others agree

Reply:
Ah amazing. This is exactly the type of discussion I wanted this PR to raise.

Hmm this item does not fit in with the issues I've opened in napari, but I am not sure if it warrants it's own issue. Let's see what a few others think and we'll go from there.

ping @jni @psobolewskiPhD (please ping any others )

Reply:
I gotta be honest I don't have a strong opinion here and could be swayed either way...

One way to think about this: we can always add the feature for plugins later, but removing it once it's been added will be super tricky. 😅

Reply:
I've updated this to say that single mods are only allowed for users but we are open to allowing it for plugins given demand.